src/ansys/acp/core/_tree_objects/_grpc_helpers/linked_object_list.py

```diff
@@ -34,11 +34,11 @@
 from ansys.api.acp.v0.base_pb2 import ResourcePath
 
 from .._object_cache import ObjectCacheMixin, constructor_with_cache
-from ..base import CreatableTreeObject, TreeObject
+from ..base import TreeObject, TreeObjectBase
```

Comment:
Small fixes to the type annotations, since the `SolidElementSet` is a read-only tree object, but still present in a linked object list.